appveyor.yml

```diff
@@ -8,9 +8,6 @@ environment:
     - PYTHON: "C:\\Python36"
       TOXENV: "docs"
 
-    - PYTHON: "C:\\Python27"
```

Comment:
We should update our cookiecutter to remove this dependency

Reply:
:+1: